Sources/GRPC/Version.swift

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2021, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+internal struct Version {
+  /// The major version.
+  internal static let major = 1
+
+  /// The minor version.
+  internal static let minor = 0
+
+  /// The patch version.
+  internal static let patch = 0
```

Comment:
An extra question: should this also contain any version qualifiers, e.g. `-alpha24`?

Reply:
I'm being optimistic about what the next version is!